app/controllers/search_controller.rb

```diff
@@ -1,5 +1,5 @@
 class SearchController < ApplicationController
   def new
-    @results = Resource.search(params[:query])
+    @results = params[:query].present? ? Resource.search(params[:query]) : []
```

Comment:
Got it. OK, yeah I suspected that it would need to be something existing outside of the controller.